app/views/layouts/_tag_edit_items.html.haml

```diff
@@ -2,7 +2,117 @@
 %div
   %h3
     - t = {:amount => @tagitems.length, :object => ui_lookup(:model => @edit[:tagging]), :objects => ui_lookup(:models => @edit[:tagging])}
-    = n_("%{amount} %{object} Being Tagged", "%{amount} %{objects} Being Tagged", @tagitems.length) % t
+    - case t[:object]
```

Comment:
Doing the `case` based on `t[:object]` is a bit volatile. The model display name can change and this switch will be broken. You should do the switch based on `@edit[:tagging]` and the actual class / model name.

Reply:
done